library/src/main/java/com/novoda/downloadmanager/lib/DatabaseHelper.java

```diff
@@ -24,8 +24,7 @@
 
     private static final String VERSION_ONE_TO_VERSION_TWO_MIGRATION_SCRIPT = "ALTER TABLE "
             + DownloadContract.Batches.BATCHES_TABLE_NAME
-            + " ADD "
-            + DownloadContract.Batches.COLUMN_HAS_STARTED + " BOOLEAN NOT NULL DEFAULT 0;";
+            + ";";
```

Comment:
Migration covered here https://github.com/novoda/download-manager/commit/92c7b0823ccaa309448fa4b6f50622b05cb22538 there is no alter with removing columns, we have to drop and recreate